compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -451,13 +508,86 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         }
 
         debug!("For closure={:?}, min_captures={:#?}", closure_def_id, root_var_min_capture_list);
+    }
 
-        if !root_var_min_capture_list.is_empty() {
-            self.typeck_results
-                .borrow_mut()
-                .closure_min_captures
-                .insert(closure_def_id, root_var_min_capture_list);
+    /// Figures out the list of root variables (and their types) that aren't completely
+    /// captured by the closure when `capture_disjoint_fields` is enabled and drop order of
+    /// some path starting at this root variable **might** be affected.
+    ///
+    /// The output list would include a root variable if:
+    /// - It was would have been moved into the closure when `capture_disjoint_fields` isn't
```

Comment:
`It would have been moved into the closure when `capture_disjoint_fields` wasn't ...`